src/plugins/expressions/common/util/create_detault_instector_adapters.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { RequestAdapter } from '../../../inspector/common';
```

Comment:
Probably some typo issue here: should it be `create_default_inspector_adapters.ts`?

Reply:
I'm not sure the history of that bar or what design consistency we need to adhere to but as a user I wouldn't necessarily look for "Download" as an option under "Inspect". If we can keep it I think it'll get more usage

Reply:
@ghudgins Let me add my 2 cents. I think if I had known about this earlier, I would have removed the `CVS Download` function during the preparation of this PR. I agree with @dej611 and It seems to me that now this is not needed. We have a common approach across Kibana that we should support. And it works through the `Inspect` panel.

About `Download all csv's` - what if that feature will be implemented  directly in `Inspect` panel. Lens is the first visualization which work with multiple `Datatables`. I think this is the reason why no one has done this before.

On the other hand, we can leave it as it is, I do not insist, maybe sometime later we will return to this....

Reply:
Tested the PR and the `Download CSV` in the Inspect panel has multiple files support already, so no difference from the Lens one.

I have no strong opinion here, there's some overlap but I find it useful anyway to have the download shortcut.